README.md

```diff
@@ -175,17 +175,41 @@ git clone git@github.com:GSA/notifications-utils.git
 
 Now go into the project directory (`notifications-utils` by default), create a
 virtual environment, and set the local Python version to point to the virtual
-environment (assumes version Python `3.9.18` is what is installed on your
+environment (assumes version Python `3.12.2` is what is installed on your
 machine):
 
 ```sh
 cd notifications-utils
-pyenv virtualenv 3.9.18 notify-utils
+pyenv virtualenv 3.12.2 notify-utils
 pyenv local notify-utils
 ```
 
 _If you're not sure which version of Python was installed with `pyenv`, you can check by running `pyenv versions` and it'll list everything available currently._
 
+## Switching to different environment
+
+Once all of pre-requisites for the project are installed and to switch to newer environment with newer python version follow the below steps to create new virtual environment.
+
+First install the newer Python version we need with `pyenv`, (say the planned upgrade to 3.15) like so :
```

Comment:
Since 3.12 is the latest version, let's keep these examples working with real versions to make it easy. :-)

```suggestion
First install the newer Python version we need with `pyenv`, (say the planned upgrade to 3.12) like so :
```

Reply:
The latest is 3.12 but as we are updating the "First-Time Project Setup" section to 3.12 ,thought this would be confusing if we have the same versions number in 'Switching to different environment' section

Reply:
Oh, I see what you mean!  We can adjust this language a bit to account for that then, where it's both for the first time setup and then "if you're upgrading an existing project setup..."

Reply:
@samathad2023 I made some additional tweaks to this language after running through the steps myself.  We could simplify it a bit more and make it clear it's just in the case of a Python upgrade.

Thank you for figuring out the process and documenting all of the details!  It made it easy to refine and finalize. 🙂